### PR TITLE
A WA for one GPU hang issue for hevc decoder

### PR DESCRIPTION
**Pull request by TianmiChen**
This is a WA solution for one GPU hang issue for
android.security.cts.StagefrightTest#testBug_63045918.

Tracked-On: OAM-97311
Signed-off-by: Chen, Tianmi <tianmi.chen@intel.com>

### CODE REVIEW

File: omx_components/src/mfx_omx_vdec_component.cpp [hunk 1]
```diff
@@ -2216,6 +2216,11 @@ mfxStatus MfxOmxVdecComponent::InitCodec(void)
                     m_MfxVideoParams.mfx.FrameInfo.FourCC = MFX_FOURCC_P010;
             }
         }
+        // A WA for one GPU hang issue for android.security.cts.StagefrightTest#testBug_63045918
+        if (m_bUseSystemMemory && (MFX_HW_TGL_LP == m_pDevice->GetPlatformType()) && (MFX_CODEC_HEVC == m_MfxVideoParams.mfx.CodecId))
```
Comment (dyang23) [inline omx_components/src/mfx_omx_vdec_component.cpp:2220]: Is this mean all hevc decode using system memory will not be supported?  Will it impact transcode and CTS?

**Review by dyang23 — APPROVED** [review]
LGTM, 
thanks

**Review by YuanjunHuang — COMMENTED** [review]
LGTM.

**Review by Shao-Feng — APPROVED** [review]
LGTM

**Pull request merged**